### DISCUSSION TITLE
How to exclude a specific subdomain from the tenant?

### DISCUSSION THREAD

**Discussion by lucasantoniooficial** [spatie/laravel-multitenancy#324] [Q&A]
I am using the package to create a multi-tenant application with a single database, but I will have two subdomains that will not be part of the multi tenancy.

I would like to know what would be the suggestion to solve this case.

I've already tried to create a separate route file and in the RouteServiceProvider set that file to the specific subdomain, but it still falls into the tenant.

I need a light.

**Comment by masterix21** [comment]
What kind of connection should these use? Landlord?

**Reply by lucasantoniooficial** [reply]
LandLord
